ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -10,9 +10,7 @@ Resource          ../../../tests/Resources/Page/DistributedWorkloads/Distributed
 
 *** Variables ***
 ${CODEFLARE_RELEASE_ASSETS}     %{CODEFLARE_RELEASE_ASSETS=https://github.com/opendatahub-io/distributed-workloads/releases/latest/download}
-${NOTEBOOK_IMAGE_STREAM_NAME}   %{NOTEBOOK_IMAGE_STREAM_NAME=s2i-generic-data-science-notebook}
-${NOTEBOOK_ADMIN_NAME}          ${TEST_USER_2.USERNAME}
```

Comment:
It seems not related to the image. Have you tested it?

Reply:
Yes, I have tested this PR locally.. Works as expected !

![image](https://github.com/user-attachments/assets/67d6ac21-515d-43d0-9e3d-5f04613d1feb)